setup.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+
+try:
+    from setuptools import setup, find_packages
+except ImportError:
+    from distutils.core import setup
+
+
+with open('README.rst') as readme_file:
+    readme = readme_file.read()
+
+with open('requirements.txt') as req_file:
+    requirements = req_file.read().split('\n')
+
+with open('requirements-dev.txt') as req_file:
+    requirements_dev = req_file.read().split('\n')
+
+with open('VERSION') as fp:
+    version = fp.read().strip()
+
+setup(
+    name="vumi-http-proxy",
+    version=version,
+    description="HTTP Proxy in Python Twisted to prevent unauthorized access to blacklisted ips",
+    long_description=readme,
+    author="Carla Wilby",
+    author_email='thisiscarlawilby@gmail.co.za',
```

Comment:
We usually put
```python
author='Praekelt Foundation',
    author_email='dev@praekeltfoundation.org',
```
here so that if people leave (or go on leave) they don't get random support requests. :)